bats/fb-test-backup.bats

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+
+@test "perform backup" {
+  mkdir -p /var/tmp/foreman-backup
+  # this is a workaround for a check that is only relevant for DB dumps
```

Comment:
I recall there was a BZ for this in downstream. Do we have an upstream issue link we can track and put in a comment here?

Reply:
I can do the :woman_dancing:

Reply:
done